docs/commons/spark.md

```diff
@@ -52,56 +51,40 @@ Docker it is typical to run a single process in a single container. Here we
 show how to start a number of workers, a single Spark master that acts as a
 coordinator (cluster manager), and submit a job.
 
-First, create a new Docker network. Let's call it spark-net. If we are running
-on a single machine, a bridge network will do.
-
-    $ docker network create --driver bridge spark-net
-
-We need to create this new network because Docker does not support automatic
-service discovery on the default bridge network.
-
 Start a Spark master:
 
-    $ docker run -dP --net spark-net --hostname spark-master --name spark-master cloudsuite/spark master
+    $ docker run -dP --net host --name spark-master cloudsuite/spark:2.4.5 master
 
 Start a number of Spark workers:
 
-    $ docker run -dP --net spark-net --name spark-worker-01 cloudsuite/spark worker spark://spark-master:7077
-    $ docker run -dP --net spark-net --name spark-worker-02 cloudsuite/spark worker spark://spark-master:7077
+    $ docker run -dP --net host --name spark-worker-01 cloudsuite/spark:2.4.5 worker spark://SPARK-MASTER-IPADDRESS:7077
+    $ docker run -dP --net host --name spark-worker-02 cloudsuite/spark:2.4.5 worker spark://SPARK-MASTER-IPADDRESS:7077
     $ ...
 
-We can monitor our jobs using Spark's web UI. Point your browser to MASTER_IP:8080, where:
-
-    $ MASTER_IP=$(docker inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' spark-master)
+We can monitor our jobs using Spark's web UI. Point your browser to SPARK-MASTER-IPADDRESS:8080, where SPARK-MASTER-IPADDRESS is the IP of the VM/host on which spark master is running. Use spark://localhost:7077 if running on the same instance.
 
 Finally, to submit a job, we can use any of the methods described in the Single
 Container section, with the addition of the network argument to Docker and
 spark-master argument to Spark.
 
 Start Spark container with bash and run spark-submit inside it to estimate Pi:
 
-    $ docker run -it --rm --net spark-net cloudsuite/spark bash
-    $ /opt/spark-1.5.1/bin/spark-submit --class org.apache.spark.examples.SparkPi \
-        --master spark://spark-master:7077 \
-        /opt/spark-1.5.1/lib/spark-examples-1.5.1-hadoop2.6.0.jar 100
+    $ docker run -it --rm --net host:: cloudsuite/spark:2.3.1 bash
+    $ /opt/spark-2.3.1/bin/spark-submit --class org.apache.spark.examples.SparkPi \
+        --master spark://SPARK-MASTER-IPADDRESS:7077 \
+        /opt/spark-2.3.1/examples/jars/spark-examples_2.11-2.3.1.jar 100
 
 Start Spark container with "submit" command to estimate Pi:
 
-    $ docker run --rm --net spark-net cloudsuite/spark submit --class org.apache.spark.examples.SparkPi \
-        --master spark://spark-master:7077 \
-        /opt/spark-1.5.1/lib/spark-examples-1.5.1-hadoop2.6.0.jar 100
+    $ docker run --rm --net host cloudsuite/spark:2.3.1 submit --class org.apache.spark.examples.SparkPi \
```

Comment:
Update version

Reply:
`examples/jars/*.jar` is not present as we are downloading only the spark binaries from [link](https://archive.apache.org/dist/spark/spark-2.4.5/spark-2.4.5-bin-hadoop2.7.tgz) 
Should I remove the lines containing SparkPi examples?